test-case/check-pause-release.sh

```diff
@@ -3,16 +3,16 @@
 set -e
 
 ##
-## Case Name: check-pause-resume
```

Comment:
Could we have a symbolic link for old check-pause-resume.sh and multiple-pause-release.sh ? Given we have no standard runner, we will have lot of custom scripts (and bug descriptions referring to how to run check-pause-resume.sh with certain parameters) . It would be ease the migration...